sormas-e2e-tests/src/test/java/org/sormas/e2etests/entities/services/TravelEntryService.java

```diff
@@ -62,4 +62,23 @@ public TravelEntry buildGeneratedEntryDE() {
         .pointOfEntryDetails("Automated test dummy description")
         .build();
   }
+
+  public TravelEntry buildGeneratedEntryWithPointOfEntryDetails(String pointOfEntry) {
```

Comment:
mention also that is for DE to differenciate them

Reply:
good point